src/Track.js

```diff
@@ -23,6 +23,7 @@ export default class {
     constructor() {
 
         this.name = "Untitled";
+        this.id = 'track'
```

Comment:
This should not be set or randomized by default to avoid multiple ids with the name content on the page if people don't want to set them.